src/Controller/Hello.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\ControllerTrait;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class Hello
+{
+    use ControllerTrait;
+
+    /**
+     * @Route("/", name="homepage")
+     */
+    public function __invoke(Request $request): Response
```

Comment:
Now my proposal is in progress: [[Routing][DX] Add full route definition for invokable controller/class](https://github.com/symfony/symfony/pull/21723)